src/app/api/images/og/delegate/route.tsx

```diff
@@ -0,0 +1,131 @@
+// @ts-nocheck
```

Comment:
It's better to use `ts-ignore` on specific sections rather than ignore the whole file. You might not be catching other ts checks this way